### ISSUE TITLE
Firefox 59 fixes and headerbar bookmark item hover styling

### ISSUE THREAD

**Issue by smithfred** [kurogetsusai/firefox-gnome-theme#28]
See today's commit to my layer: https://github.com/smithfred/firefox-gnome-theme-3.26-layer

I haven't had the opportunity to convert to a fork yet sorry, but the changes are pretty straightforward.

**Comment by lunakurame** [comment]
I merged your changes:
- this part doesn't seem to do anything on my setup, so I ignored it:
	```css
	/* Fix tab close icon padding and opacity (no longer contains icon element) */
	:root {
	    --gnome-tab-close-icon-filter: opacity(0.25);
	}
	.tab-close-button {
	    filter: var(--gnome-tab-close-icon-filter);
	    padding: 3px;
	}
	```
- this part seems to be specific to your repo only, I don't even have that variable:
	```css
	:root {
	    --gnome-popover-arrow-filter: none;
	}
	```
- this part is nice so I added it to my repo:
	```css
	/* Bookmark toolbar items on nav. bar */
	#nav-bar toolbarbutton.bookmark-item {
	    border: 1px solid transparent !important;
		border-radius: 3px !important;
		height: 34px !important;
		margin: 6px 3px !important;
		padding: 0 4px !important;
	}
	#nav-bar toolbarbutton.bookmark-item:not([disabled]):hover {
		border: var(--gnome-headerbar-button-border) !important;
		border-bottom: var(--gnome-headerbar-button-border-bottom) !important;
		background-image: var(--gnome-headerbar-button-hover-bgimage);
		box-shadow: var(--gnome-headerbar-button-box-shadow);
	}
	#nav-bar toolbarbutton.bookmark-item:not([disabled]):active {
		border: var(--gnome-headerbar-button-border) !important;
		border-bottom: var(--gnome-headerbar-button-border-bottom) !important;
		background-image: var(--gnome-headerbar-button-active-bgimage);
		box-shadow: var(--gnome-headerbar-button-active-box-shadow);
	}
	```

Tested on FF 59 stable from Mozilla's FTP (seems like they removed CSD support from that version), an old FF 59 alpha (which supported CSD) and Fedora 27 + recent FF 59, seems okay to me, including CSD.

Also this repo finally is up to date (FF 59 stable + an experimental CSD patch for FF 60+), so you can check if there's still anything which is broken in my repo but fixed in your one.

**Comment by lunakurame** [comment]
Oh, I see what is the first part supposed to do, it's a fix for tab close buttons. It was already fixed in #31, except it still has wrong opacity, I'll push a fix in a second.
